src/index.tsx

```diff
@@ -780,6 +781,8 @@ class RichMarkdownEditor extends React.PureComponent<Props, State> {
         justify="center"
         dir={dir}
         column
+        spellCheck={typeof this.props.spellCheck === 'boolean' ? 
```

Comment:
do we need a typeof check? cant we just do `!!this.props.spellCheck`?

Reply:
So this ensures that the default value of spellCheck is true.

If we had `!!undefined` -> that would evaluate as false. 

So the only way to turn off spellCheck would be to specify is as `false`